monai/apps/deepedit/transforms.py

```diff
@@ -595,20 +595,20 @@ def __call__(self, data: Mapping[Hashable, np.ndarray]) -> Dict[Hashable, np.nda
                     pass
                 else:
                     keep_guidance.append(aux_label)
-                    counter = counter + len(self.tmp_guidance[aux_label])
+                    counter = counter + len(self.guidance[aux_label])
                     # If collected clicks is bigger than max clicks, discard the others
                     if counter >= num_clicks:
                         for key_label in d["label_names"].keys():
                             if key_label not in keep_guidance:
-                                self.tmp_guidance[key_label] = []
+                                self.guidance[key_label] = []
                         logger.info(f"Number of simulated clicks: {counter}")
                         break
 
                 # Breaking once all labels are covered
                 if len(keep_guidance) == len(d["label_names"].keys()):
                     logger.info(f"Number of simulated clicks: {counter}")
                     break
-
+        d[self.guidance_key] = self.guidance
```

Comment:
I'd suggest we first perform training and inference on MONAI Label. This might break the training

Reply:
![image](https://user-images.githubusercontent.com/14771322/186558021-931f8f0f-f214-4c7b-8ce9-1422c26ec839.png)

this is from training

Reply:
![image](https://user-images.githubusercontent.com/14771322/186558136-4ba0a6ea-3712-4b64-abd4-c42d3560d0a8.png)

this is from monai label inference, when inference, AddRandomGuidanceDeepEditd is not used.
AddRandomGuidanceDeepEditd only used in training to add some random simulation guidance

Reply:
![image](https://user-images.githubusercontent.com/14771322/186559627-26aef0ce-2a02-4b0f-9407-2f7214d39405.png)

this is from monai label deepedit training, same as train in monai